src/firebolt_ingest/table_service.py

```diff
@@ -18,11 +18,15 @@
 )
 from firebolt_ingest.utils import format_query
 
-EXTERNAL_TABLE_PREFIX = "ex_"
-
 
 class TableService:
-    def __init__(self, table: Table, connection: Connection):
+    def __init__(
+        self,
+        table: Table,
+        connection: Connection,
+        ext_prefix: str = "ex_",
+        int_prefix: str = "",
```

Comment:
I feel like int can be confused with integer here, let's have it `external_prefix` and `internal_prefix`. Also please add those params to the docstring on L37.

Reply:
done